doc/news/changes/minor/20180728GiovanniAlzettaTimoHeister

```diff
@@ -0,0 +1,3 @@
+Corrected potential bug: VectorTools::interpolate now checks for weights to be positive.
+<br>
+(Giovanni Alzetta, Timo Heister, 2018/07/24)
```

Comment:
At least, this is not accurate anymore. Now, we just set the values for all the components we know about.